File: ABC081A/placing-marbles.go
/*
ABC081A - Placing Marbles
https://atcoder.jp/contests/abs/tasks/abc081_a
*/

package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

var sc = bufio.NewScanner(os.Stdin)

func readLine() string {
	if !sc.Scan() {
		return ""
	}
	return sc.Text()
}

func main() {
	nums := strings.Split(readLine(), "")

	var result int
	for _, num := range nums {
		if num == "1" {
			result++
		}
	}

	fmt.Printf("%d\n", result)
}

File: ABC081A/placing-marbles.cpp
/*
ABC081A - Placing Marbles
https://atcoder.jp/contests/abs/tasks/abc081_a
*/

#include <iostream>
#include <string>
using namespace std;
int main() {
    cin.tie(0);
        ios::sync_with_stdio(false);

    string s;
    int n;

    cin >> s;
    for (int i = 0; i < 3; i++)
        if(s[i] == '1') n++;

    cout << n << "\n";
}